lib/rules/no-else-return.js

```diff
@@ -12,6 +12,35 @@
 const astUtils = require("../util/ast-utils");
 const FixTracker = require("../util/fix-tracker");
 
+//------------------------------------------------------------------------------
+// Helpers
+//------------------------------------------------------------------------------
+
+const BLOCK_SCOPED_DECLARATION = /^(?:VariableDeclarator|ClassDeclaration)$/;
+
+/**
+ *
+ * @param {*} scope  current scope
+ * @param {Token} startToken s
+ * @param {Token} endToken s
+ * @returns {boolean} s
```

Comment:
Could you please fill in these comments (this line, and the two lines above it)?